### ISSUE TITLE
Config routes with no response fail to be recognised as a route

### ISSUE THREAD

**Issue by jasdeepkhalsa** [cypress-io/cypress#3940]
### Current behavior:

If we try to create the following route:

```javascript
cy.route({
  method: 'GET',
  url: '**/some-route',
  status: 400,
}).as('someRoute');
```

This is failed to be picked up by Cypress as a valid route:

```javascript
cy.wait('@someRoute'); // The route is not recognised, so this assertion will fail
```

The current workaround for this is to provide an empty response object, which is technically not correct behaviour, as this particular route does not return any response (not even an empty JSON payload):

```javascript
cy.route({
  method: 'GET',
  url: '**/some-route',
  status: 400,
 response: {}
}).as('someRoute');
```

### Desired behavior:

Routes without responses should be able to be configured / considered as valid routes by Cypress:

```javascript
cy.route({
  method: 'GET',
  url: '**/some-route',
  status: 400,
}).as('someRoute');
```

### Steps to reproduce: (app code and test code)

Same as per the [current behaviour](#current-behavior)

### Versions

Cypress: Latest, 3.2.0
Operating System: macOS High Sierra 10.13.6
Browser: Chrome and/or Electron 59

**Comment by jeep87c** [comment]
I'm having the exact same issue. It also fails doing something like `cy.route('/some/route')`

**Comment by jpierson-at-riis** [comment]
Is this something that will still be done as in the new `intercept` command seems to be replacing `route` anyway? Is this still a problem with `intercept`? I'll likely be upgrading a project sometime in the next few months and I want to gauge whether there is an available solution now with `route` or whether we can include the solution to the empty response problem another reason to upgrade and to adopt `intercept`.

**Comment by jpierson-at-riis** [comment]
By the way, there is some evidence of others running into this issue as well.
https://stackoverflow.com/a/61618367/83658

**Comment by jpierson-at-riis** [comment]
Ok, this looks like a duplicate to me of https://github.com/cypress-io/cypress/issues/8018 which was supposed to be fixed in v4.11.0. Our project where we were running into this issue was v4.6.0 and testing with v4.11.0 does in fact seem to correct this behavior. I see when we pass empty string `""` as the response value that it is treating the route as stubbed.

**Comment by emilyrohrbough** [comment]
In the upcoming v12.0.0 release, the deprecated `cy.route()` and `cy.server()` commands and the deprecated `Cypress.Server.defaults` API will be removed. This has been replaced with [`cy.intercept()`](https://docs.cypress.io/api/commands/intercept). Close as this is no longer application.

Please try recreating this issue with the latest Cypress version and `cy.intercept()`. If you are still having problems please open a new Github issue.
